test/application.js

```diff
@@ -27,7 +31,11 @@ describe('Application', () => {
 
     it ("doesn't set activeACL to document referrer if not in ACL", () => {
       const insecureApp = new Application();
-      global.document = {referrer: 'http://evilsite.com', createElement: document.createElement.bind(document), body: { addEventListener: () => console.log('mock addEventListener') } };
+      global.document = {
+        referrer: 'http://evilsite.com',
+        createElement: document.createElement.bind(document),
+        addEventListener: () => console.log('mock addEventListener'),
+        body: { addEventListener: () => console.log('mock addEventListener') } };
```

Comment:
nit: putting the last end bracket `}` on a separate line will make this format more consistent with other [objects declared](https://github.com/cerner/xfc/blob/11060b7cfb2293aeeeffee3706e69f932ea18fbd/test/application.js#L121-L125)